Ports/.port_include.sh

```diff
@@ -68,6 +68,13 @@ fetch() {
         IFS=$OLDIFS
         read url filename auth_sum<<< $(echo "$f")
         echo "URL: ${url}"
+
+        # FIXME: Serenity's curl port does not support https, even with openssl installed.
```

Comment:
Failing this, you _could_ try using `pro`

Reply:
pro doesn't have the ciphers in common with ftp.gnu.org, so it's not a great solution either. I've added it as a fallback when curl isn't installed though.

Reply:
shame.
Eventually it will have _some_ modern ciphersuite...eventually.